app/browser/api/ledger.js

```diff
@@ -559,9 +561,14 @@ const getPublisherData = (result, scorekeeper) => {
     pinPercentage: result.pinPercentage,
     weight: result.pinPercentage
   }
-  // HACK: Protocol is sometimes blank here, so default to http:// so we can
-  // still generate publisherURL.
-  data.publisherURL = (result.protocol || 'http:') + '//' + result.publisherKey
+
+  data.publisherURL = result.publisherURL || (result.protocol || 'http:') + '//' + result.publisherKey
```

Comment:
would be nice to use parens to make this clear-  I think you're meaning it to be:
`result.publisherURL || ((result.protocol || 'http:') + '//' + result.publisherKey)`

Reply:
done